### PR TITLE
devfsadm: add French translation

### PR DESCRIPTION
**Pull request by jay-tau**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/sunos/devfsadm.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# devfsadm
+
+> Commande d'administration pour `/dev`. Maintient le `/dev` espace de noms.
+> Plus d'information : <https://www.unix.com/man-page/sunos/1m/devfsadm>.
+
+- Rechercher de nouveaux disques:
```
Comment (kbdharun) [inline pages.fr/sunos/devfsadm.md:6]:
```suggestion
- Rechercher de nouveaux disques :
```

File: pages.fr/sunos/devfsadm.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`devfsadm -c disk`
+
+- Nettoyez tout pendaison /dev liens et rechercher un nouvel appareil:
```
Comment (kbdharun) [inline pages.fr/sunos/devfsadm.md:10]:
```suggestion
- Nettoyez tout pendaison /dev liens et rechercher un nouvel appareil :
```

File: pages.fr/sunos/devfsadm.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`devfsadm -C -v`
+
+- Marche à sec - sortir ce qui serait changé mais ne faire aucune modification:
```
Comment (kbdharun) [inline pages.fr/sunos/devfsadm.md:14]:
```suggestion
- Marche à sec - sortir ce qui serait changé mais ne faire aucune modification :
```

File: pages.fr/sunos/devfsadm.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# devfsadm
+
+> Commande d'administration pour `/dev`. Maintient le `/dev` espace de noms.
+> Plus d'information : <https://www.unix.com/man-page/sunos/1m/devfsadm>.
+
+- Rechercher de nouveaux disques :
+
+`devfsadm -c disk`
+
+- Nettoyez tout pendaison /dev liens et rechercher un nouvel appareil :
```
Comment (kbdharun) [inline pages.fr/sunos/devfsadm.md:10]:
```suggestion
- Nettoyez tout pendaison `/dev` liens et rechercher un nouvel appareil :
```
Just a small suggestion.

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**